exporters/otlp/common/src/jmh/java/io/opentelemetry/exporter/internal/otlp/LogsRequestMarshalerBenchmark.java

```diff
@@ -84,22 +84,21 @@ public class LogsRequestMarshalerBenchmark {
         .setSeverityText("INFO")
         .emit();
 
-    SdkEventLoggerProvider eventLoggerProvider = SdkEventLoggerProvider.create(loggerProvider);
-    EventLogger eventLogger = eventLoggerProvider.get("event-logger");
-    eventLogger
-        .builder("namespace.my-event-name")
-        // Helper methods to set primitive types
-        .put("stringKey", "value")
-        .put("longKey", 1L)
-        .put("doubleKey", 1.0)
-        .put("boolKey", true)
-        // Helper methods to set primitive array types
-        .put("stringArrKey", "value1", "value2")
-        .put("longArrKey", 1L, 2L)
-        .put("doubleArrKey", 1.0, 2.0)
-        .put("boolArrKey", true, false)
-        // Set complex data
-        .put("key", Value.of(Collections.singletonMap("childKey1", Value.of("value"))))
```

Comment:
`EventBuilder` had `put` overloads which add KeyValue entries to the log record value. Producing the equivalent with `LogRecordBuilder` is more cumbersome.

Reply:
I think this may be ok now that we're deprioritizing event bodies

Reply:
`LogRecordBuilder` also lacks [overloads for setting attributes](https://github.com/open-telemetry/opentelemetry-java/blob/main/api/all/src/main/java/io/opentelemetry/api/logs/LogRecordBuilder.java#L87-L102) which we see in the [span api](https://github.com/open-telemetry/opentelemetry-java/blob/main/api/all/src/main/java/io/opentelemetry/api/trace/Span.java#L87-L189).

Obviously we'd want to add these as part of an ergonomic / user facing API.

Reply:
> I think this may be ok now that we're deprioritizing event bodies

Until we land #6983 (or equivalent) the body is the only place you can put complex data as a java user.

Reply:
> `LogRecordBuilder` also lacks [overloads for setting attributes](https://github.com/open-telemetry/opentelemetry-java/blob/main/api/all/src/main/java/io/opentelemetry/api/logs/LogRecordBuilder.java#L87-L102) which we see in the [span api](https://github.com/open-telemetry/opentelemetry-java/blob/main/api/all/src/main/java/io/opentelemetry/api/trace/Span.java#L87-L189).
> 
> Obviously we'd want to add these as part of an ergonomic / user facing API.

I'd probably be in favor of adding these overloads in the LogRecordBuilder regardless of the future user facing API, but I don't think there's any rush on this